website/src/blogs/containerization-meetup-containers-for-storage-too.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Containerization meetup: Containers for storage too"
+author: Uma Mukkara
+author_info: Contributor at openebs.io, Co-founder& COO@MayaData. Uma led product development in the early days of MayaData (CloudByte).
```

Comment:
```suggestion
author_info: Contributor at openebs.io, Co-founder & COO@MayaData. Uma led product development in the early days of MayaData (CloudByte).
```

Reply:
we can remove `""` from title. Originally it was not present